lib/pundit/matchers/utils/only_actions/error_message_formatter.rb

```diff
@@ -14,8 +14,10 @@ module OnlyActions
         module ErrorMessageFormatter
           def message
             "#{policy_name} expected to have only actions #{matcher.expected_actions} #{expected_kind}, but " \
-            "#{"#{mismatches_are(missed_expected_actions)} #{opposite_kind} and " unless missed_expected_actions.empty?}" \
-            "#{mismatches_are(unexpected_actions)} #{expected_kind} too"
+              "#{unless missed_expected_actions.empty?
+                   "#{mismatches_are(missed_expected_actions)} #{opposite_kind} and "
+                 end}" \
+              "#{mismatches_are(unexpected_actions)} #{expected_kind} too"
```

Comment:
the only change to production code. Only needed because of line length. We can increase to 140 chars

Reply:
I would prefer to keep using short line lengths (80 to 100 characters) for folks who use a split view in their code editor. Reasoning elaborated here: https://rubystyle.guide/#max-line-length